app/core/initialize.js

```diff
@@ -39,6 +41,32 @@ if (utils.isOzaria) {
   Vue.directive('mask', VueMaskDirective);
 }
 
+datadogRum.init({
+  ...(utils.isCodeCombat ? {
+    applicationId: '0fe8c7ec-5984-4191-b5f3-666fa6617477',
+    clientToken: 'pubb0ab7e396292808398f9cccbd7cdde42',
```

Comment:
no harm in putting them in public? Can we specify in datadog from where it can receive traffic so that someone else can't use it on their site? Though it's relatively non-critical so should be fine for now.

Reply:
@mrfinch I think it must be public. If it's not public, the client won't see it.